packages/grafana-prometheus/src/querybuilder/components/LabelParamEditor.tsx

```diff
@@ -29,7 +30,7 @@ export function LabelParamEditor({
       openMenuOnFocus
       onOpenMenu={async () => {
         setState({ isLoading: true });
-        const options = await loadGroupByLabels(query, datasource);
+        const options = await loadGroupByLabels(timeRange, query, datasource);
```

Comment:
I'd prefer `timeRange` as the 3rd parameter. This is mostly a personal preference but I believe it creates less mental overheat. Like we load the labels fro that query from this datasource in this time range.

Reply:
we can ignore this I think